### PR TITLE
[CARBONDATA-1880] Documentation for merging small files

### PR DESCRIPTION
**Pull request by sgururajshetty**
Added the documentation for merging small file for better performance.

### CODE REVIEW

**Comment by ravipesala** [comment]
Can one of the admins verify this patch?

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

File: docs/configuration-parameters.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ This section provides the details of all the configurations required for CarbonD
 | carbon.options.is.empty.data.bad.record | false | If false, then empty ("" or '' or ,,) data will not be considered as bad record and vice versa. | |
 | carbon.options.bad.record.path |  | Specifies the HDFS path where bad records are stored. By default the value is Null. This path must to be configured by the user if bad record logger is enabled or bad record action redirect. | |
 | carbon.enable.vector.reader | true | This parameter increases the performance of select queries as it fetch columnar batch of size 4*1024 rows instead of fetching data row by row. | |
+| carbon.task.distribution | merge_small_files | Setting this parameter value to *merge_small_files* will merge all the small files to a size of (128 MB). During data loading, all the small CSV files are combined to a map task to reduce the number of read task. This enhances the performance. | |
```
Comment (QiangCai) [inline docs/configuration-parameters.md]: 1. carbon.task.distribution is only for the query, not be used by data loading.
Global_Sort loading will always merge small CSV files, not require this configuration.
2. better to list all values of carbon.task.distribution
custom, block(default), blocklet, merge_small_files

**Pull request closed without merging**